test/remote/run_remote/run_remote.go

```diff
@@ -103,8 +108,56 @@ func main() {
 	flag.Parse()
 	suite = remote.InitE2ERemote()
 
-	if *serviceAccount == "" {
-		glog.Fatal("You must specify a service account to create an instance under that has at least OWNERS permissions on disks and READER on instances.")
+	if *runInProw {
```

Comment:
hummm why not just check if project is empty?

Reply:
I'd rather there be a special flag for running in prow since there's a lot of special set-up involved. If the project is empty and runInProw is not set then the user gets a nice reasonable error to set the project instead of something that mentions `Boskos` and other things they dont need to know about